crypto/pkcs12/p12_sbag.c

```diff
@@ -128,7 +128,6 @@ PKCS12_SAFEBAG *PKCS12_SAFEBAG_create_secret(int type, int vtype, const unsigned
     case V_ASN1_OCTET_STRING:
         {
             ASN1_OCTET_STRING *strtmp = ASN1_OCTET_STRING_new();
-
```

Comment:
Why did you delete this empty line? An empty line after declaration is required by the coding style guidelines.

Reply:
Fixed now